posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+number for the pixels of margins and sizes was a challenge. I tried several 
+values before finding one that closely matched the proposed mock-up only to 
+realize later that “pixels” was not the best unit for the specifications. I 
+ended up using “em” since it was more consistent across operating systems.
+Isabela, Stephanie and Carlos were part of this process as well. Between the 
+four of us we managed to implement all the changes that we had planned for 
+Spyder 5, the new color palette, the complete redesign of toolbars and the 
+new set of icons. It was an arduous task, more than we all expected, but at 
+the end we were all very happy with the results and thankful to Isabela for 
+helping us to give a new face to Spyder. 
+
+## What did we get?
+
+**[Isabela]** Individually, the colors, toolbars, and icons may feel like 
+small adjustments, but those are some of the elements that make up most of 
+Spyder.  When they are together, those small adjustments set the mood in the 
+interface; they are more noticeable, and rooted in the Spyder UI many people 
+are already familiar with. While the changes may feel obvious when they are 
+new, they are also chosen to create consistent patterns across interactions 
+that can become more comfortable over time. Spyder’s default dark and light 
+modes, for example, used to use a different set of UI elements between modes. 
+Now they both use the same elements and it is only the colors that change. 
+This makes it easier for users to jump into a familiar interface and take 
+what they know from working in one space to another. For contributors, it 
+gives a more clear UI pattern for them to follow in their own work.
+
+### Before and after (Dark theme)
+
+![Spyder 5 vs Spyder 4 dark theme](/images/spyder5-comparedark.png)
+
+### Before and after (Light theme)
+
+![Spyder 5 vs Spyder 4 light theme](/images/spyder5-comparelight.png)
+
+## What did we learn :)?
+
+**[Isabela]** From developing new skills to working as a team for the first 
+time, I think we both took a lot from this process. Here are some lessons that 
+stood out to us.
+
+**[Juanita]** 
+
+- Sometimes it is better to try some of the ideas during the process, than 
+having long discussions about an idea and implementing at the end. In some 
+cases you end up realizing that things don’t look as good as you thought they 
+would, or that some are not even possible.
+- One of the most important parts of the design process is to get yourself in 
+the users’ shoes. At the end, they are the reason why we work to improve 
+things constantly.
+- Occasionally, less is more. Simple and consistent is better than crowded 
+and complicated. 
+
+**[Isabela]** 
+
+- Don’t be afraid of asking questions even when you think you understand the 
+problem because every bit of information can be useful to understanding what 
```

Comment:
```suggestion
problem because every bit of information can be useful to better grasping what 
```

Avoids saying `understand` and `understanding` twice in the same sentence.